### ISSUE TITLE
Training on SD 1.5

### ISSUE THREAD

**Issue by shenzy08** [drboog/ProFusion#8]
Hello, I would like to know that whether I can train this model on stable-diffusion-v1-5, which is applicable to ControlNet. I have tried to change the base model, but encounter the dimension mismatch error, and it cannot be addressed simply like changing the clip model. Please offer me some help, thank you!

**Comment by drboog** [comment]
Yes, you need to change the clip model, and also change some dimension arguments. 
For example, because SD v1.5 and SD v2 are based on different text encoders, you need to change the [output dimension](https://github.com/drboog/ProFusion/blob/87d27607c7b47b6222560b73566ebeee92c4f891/train.py#L510). You may also want to change the dimension of [projection](https://github.com/drboog/ProFusion/blob/87d27607c7b47b6222560b73566ebeee92c4f891/diffusers/src/diffusers/models/promptnet.py#L188) if you change CLIP image encoder.
